impeller/scene/animation/animation.cc

```diff
@@ -117,7 +117,7 @@ const std::vector<Animation::Channel>& Animation::GetChannels() const {
   return channels_;
 }
 
-Scalar Animation::GetEndTime() const {
+std::chrono::duration<Scalar> Animation::GetEndTime() const {
```

Comment:
I'd put this in `//flutter/impeller/base/timing.h` and typedef it to `SecondsF` along with other related stuff from chrono. Something [like this perhaps](https://github.com/chinmaygarde/sft/blob/f606480395e461e158acdb09a53ed83c2bd209ec/src/timing.h). Your call though. As it reads, `duration<Scalar>` is a bit unclear.

Reply:
SGTM, done.